src/lib/menu/menu-foundation.ts

```diff
@@ -16,27 +17,27 @@ export interface IMenuFoundation extends ICustomElementFoundation {
   optionsFactory: MenuOptionFactory | undefined;
   selectedIndex: number;
   selectedValue: any;
-  placement: PopupPlacement;
-  fallbackPlacements: PopupPlacement[];
+  placement: PositionPlacement;
```

Comment:
Maybe some of these common popover properties (placement, offset) could be made into a mixin?

Reply:
I thought about this, but since we intend to deprecate these components in favor of new patterns (inline/static usage) I figured I would save that until then and keep these changes focused on just list-dropdown.